src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -5511,7 +5511,9 @@ export class ZoneServer2016 extends EventEmitter {
       this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID == 12994 ||
       this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID == 9114 ||
       this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID == 9945 ||
-      this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID == 12898
+      this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID == 12898 ||
```

Comment:
we don't have enums for this ? ( i know this was done like that before )

Reply:
No, there currently aren't enums for descriptionIds